RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -152,17 +154,29 @@ class ManageSubscriptionsViewModel: ObservableObject {
             self.loadingPath = nil
         }
     }
+
+    func handleInAppBrowserDismiss() {
+        self.inAppBrowserURL = nil
+    }
 #endif
 
 }
 
+struct IdentifiableURL: Identifiable {
```

Comment:
Not sure if this is the best way but I couldn't make `URL` `Identifiable` directly without getting a warning.